src/main/kotlin/uk/gov/dluhc/printapi/database/repository/PrintDetailsRepository.kt

```diff
@@ -40,7 +40,12 @@ class PrintDetailsRepository(client: DynamoDbEnhancedClient, tableConfig: Dynamo
         return index.query(query).flatMap { it.items() }
     }
 
-    private fun key(partitionValue: String, sortValue: String): Key =
+    fun getAllByStatus(status: Status): List<PrintDetails> {
+        // TODO: This is temporary as dynamodb has given me enough headaches for now
+        return table.scan().flatMap { it.items() }.filter { it.status == Status.PENDING_ASSIGNMENT_TO_BATCH }
```

Comment:
Need a cleaner way to do this as `scan()` is potentially too inefficient.

Reply:
So that this PR can progress (and unblock related tickets) I've created a separate subtask for this: https://technologyprogramme.atlassian.net/browse/EIP1-2351